### ISSUE TITLE
Custom invoice creation pushes next invoice day

### ISSUE THREAD

**Issue by thepagan** [U-CRM/billing#43]
When creating the custom invoices to cover the prorated amount from 1/2 month and then creating another invoice for this month has pushed the "Next Invoicing Day" ahead to 9/1 as seen below:

![screen shot 2016-07-06 at 1 15 21 pm](https://cloud.githubusercontent.com/assets/1947126/16625626/e003fac0-437b-11e6-83b8-8567526d5d35.png)

There is not an invoice created for 8/1/2016 - 8/30/2106 and it's set for "Forward Billing"

TLDR;

New client active from 6/15/2106 forward, created two seperate invoices for 6/15 - 6/31 and 7/1 - 7/30.  Next auto invoice creation date is 9/1 not 8/1.

**Comment by Pakket** [comment]
I have this same issue ... I want to test the auto-creating of invoices (reoccurring), but, no matter the date, it always pushed the "Next invoice" to a date that is the month after next.

**Comment by Pakket** [comment]
Also @thepagan , they instructed me to post any further issues on the UBNT forums, per this response:
http://community.ubnt.com/t5/UCRM-Complete-WISP-Management/Issue-Bug-reporting-to-continue-on-the-GitHub-page-or-in-this/m-p/1612387#U1612387

**Comment by keksa** [comment]
Should be fixed at 2.0.8

**Comment by UBNT-Petr** [comment]
fixed in v2.0.9
